src/mapper/sensor_summary_mapper.py

```diff
@@ -2,7 +2,10 @@
 
 
 def to_sensor_list(sensor_id_list):
-    return list(map(lambda id: to_sensor(id), sensor_id_list))
+    sensor_id_list = list(map(lambda id: to_sensor(id), sensor_id_list))
+    for sensor_id in sensor_id_list:
+        sensor_id = str(sensor_id)
+    return sensor_id_list
```

Comment:
the naming is a bit confusing... This line `list(map(lambda id: to_sensor(id), sensor_id_list))` returns a list of dictionary. but the name suggests it returns a list of `sensor ids`. Also, the intention of for loop is unclear.  I might have missed something. Pls clarify

Reply:
This was left behind when I was trying to fix serialization  problem , got the lines removed.